### ISSUE TITLE
Makefile build fails

### ISSUE THREAD

**Issue by nicdesousa** [open-policy-agent/opa-docker-authz#7]
The master branch build is failing with:
```
$ make
+ cd /go/src/github.com/open-policy-agent/opa-docker-authz
+ go get ./...
# github.com/open-policy-agent/opa-docker-authz
./main.go:271: not enough arguments in call to h.Handler.ServeTCP
Makefile:6: recipe for target 'build' failed
make: *** [build] Error 2
```
A similar issue was reported and resolved at:
https://github.com/ContainX/docker-volume-netshare/issues/84

**Comment by tsandall** [comment]
We can vendor the dependencies and lock to the old version of the plugin library and/or update to the latest.

**Comment by nicdesousa** [comment]
Thank you, this Issue has been resolved in v0.1.6.
